### DISCUSSION TITLE
[QUESTION]: Address copy from «Billing address» or «Shipping Address» does not work in Invoice

### DISCUSSION THREAD

**Discussion by mad99** [YetiForceCompany/YetiForceCRM#14334] [How to] [answered]
Hi,
since the upgrade from YF 5.2.0 to 6.1.0 SPv7 I have some problems to copy the Address from the «Organisation/Account».

Unfortunately the Organisation-Data seams not to copy in the form for a new invoice anymore. 
In 5.2.0 there was a button 
Address copy from «Organisation». 
![YetiForceProblem_1A](https://user-images.githubusercontent.com/6169308/110150032-09c4bf00-7ddf-11eb-8ded-0203af022ca3.png)

**This works all-time fine. But there was the Section «Primary Address».**

![YetiForceProblem_1B](https://user-images.githubusercontent.com/6169308/110150088-177a4480-7ddf-11eb-85de-7f4745767559.png)

But now, in 6.1.0 there is instead a Billing address and the Solution to "Address copy from: «Shipping address».
![YetiForceProblem_2](https://user-images.githubusercontent.com/6169308/110150391-83f54380-7ddf-11eb-9429-c83a3f3ded61.png)

So first in «Organisation/Account», I copy the «Primary Address to «Weiterleitungsadresse/Forwarding address» and also to the «Shipping address». All seams to be fine. The addresses were copied. I reload the page and the fields are still filled.

![YetiForceProblem_1](https://user-images.githubusercontent.com/6169308/110148694-6cb55680-7ddd-11eb-8d9c-fa18aa8de1d9.png)

But in the invoice form, when I try to copy the «Shipping address» in the Billing-Part, or the «Billing Address» in the Shipping-Part, I got always the errors «Keine Adresse zum kopieren/No address to copy».

Hope someone can help me/push me to the right direction, where I have to define this addresses if not from the account datas => prepared in the account-settings (see above).
Maybe there is some documentation about, unfortunately I can't find help in the Git-Issues.

Would be great to get help
Martin

**Comment by mariuszkrzaczkowski** [comment, marked as answer]
https://github.com/YetiForceCompany/YetiForceCRM/discussions/13943 https://github.com/YetiForceCompany/YetiForceCRM/commit/e533fbd5b768f68f941e4838636c79ec84d7ab93
![image](https://user-images.githubusercontent.com/10198654/110169695-5ae1ac80-7df9-11eb-99b0-6556622a6ffd.png)

**Comment by mad99** [comment]
Hi @mariuszkrzaczkowski, 
so this printScreen means, this missing «copy possibilities» is by system not from my migration process?

- Do I have to write the address in YT6.1.0SPv7 manually in the form?
- When will YT 6.2 be published? (Can't find a date in Milestones).
- 
Thanks
Martin

**Reply by apcloic** [reply]
@mad99 apply the fix and "copy from" will be back.

**Comment by mad99** [comment]
Ahh, know I see the linked-number.

I don't like to kill my Yeti. So, is it right, I can make it come back with two files changing:

1. public_html/layouts/basic/modules/Vtiger/resources/Edit.js
2. config/version.php

Like described under https://github.com/YetiForceCompany/YetiForceCRM/commit/e533fbd5b768f68f941e4838636c79ec84d7ab93

Thanks for help

**Reply by mad99** [reply]
### For others who are searching a working solution.
Have changed the THREE filesContent and the button is back.

1. public_html/layouts/basic/modules/Vtiger/resources/Edit.js
2. public_html/layouts/basic/modules/Vtiger/resources/Edit._**min**_.js
3. config/version.php


![yetiForceInvoiceOK](https://user-images.githubusercontent.com/6169308/110232643-3d0e6780-7f1f-11eb-8a9a-4f3916c3831b.png)

@apcloic: Thanks for hint
@mariuszkrzaczkowski: Thanks for help
